python/cudf/cudf/core/dataframe.py

```diff
@@ -1317,6 +1317,26 @@ def _repr_html_(self):
     def _repr_latex_(self):
         return self._get_renderable_dataframe().to_pandas()._repr_latex_()
 
+    def _get_columns_by_label(self, labels, downcast=False):
```

Comment:
I think MyPy will eventually complain about this function having a different signature from its base class...